source/index.js

```diff
@@ -211,19 +215,19 @@ class CacheableLookup {
 	}
 
 	async query(hostname) {
+		this.stats.query++;
```

Comment:
This is incorrect. `query` should be only incremented when `source` is set to `query`.

Reply:
```suggestion
```